uperf/uperf-multiplex-example.json

```diff
@@ -0,0 +1,27 @@
+{
+    "common": [
+    {
+        "name":"global",
+        "params": [
+        { "arg": "duration", "vals": [ "90" ] },
```

Comment:
Lets fix the indentation.  Since this is supposed to be an example for everyone to learn from lets make it easier to read and understand the hierarchy.